e2e-tests/IIoTPlatform-E2E-Tests/deploy/DeploymentConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.TestExtensions {
+    using System;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Common.Exceptions;
+    using System.Threading;
+    using Microsoft.Azure.Devices;
+
+    public class DeploymentConfiguration {
+
+        public DeploymentConfiguration(IIoTPlatformTestContext context) {
+            _context = context ?? throw new ArgumentNullException(nameof(context));
+        }
+
+        /// <summary>
+        /// Create or update deployment configuration
+        /// </summary>
+        /// <returns>configuration</returns>
+        /// <returns>forceUpdate</returns>
+        /// <returns>deploymentId</returns>
+        /// <returns>ct</returns>
+        public async Task<Configuration> CreateOrUpdateConfigurationAsync(
```

Comment:
A suggestion: this whole method can become part of `RegistryHelper` class. `RegistryHelper` is intended to be a helper class for managing IoT Hub device registry. And deployments are part of that registry. In that case, my comments above about only passing `RegistryManager`/`RegistryHelper` to the constructor of `DeploymentConfiguration` can be ignored,

Reply:
Addressed in #955.